auto_round/export/export_to_autoround/export.py

```diff
@@ -89,8 +90,11 @@ def dynamic_QuantLienar_for_packing(backend, bits, group_size):
         return QuantLinear
     ##export all use trition, inference use exllamav2
     elif "autoround" in backend or "auto-round" in backend or "auto_round" in backend:
-        from auto_round_extension.cuda.qliner_triton import QuantLinear
-        return QuantLinear
+        if device.type == "cpu":
+            return qlinear_qbits.QuantLinear
```

Comment:
During the packing phase, the model is always on the CPU. If the packing of qbits differs, it is better to use the standard format, as qbits are compatible with the standard. However, the reverse is not true.

Reply:
yes, your are right, i use the default triton linear when exporting model and this dosen't affect model output.